wordpress-mobile/WordPress-iOS
# Issue/10735 numeric tag fix

Fixes #10735 

To test:
 - Create a new Post on a WP.com site
 - Add some random content to it
 - Edit the Post Settings
 - Change the tags to have numbers, for example: `2010, 2011, 2012`
 - Save the post
 - Publish or Save Draft
 - Open the post
 - Update some content
 - Save/Update the post
 - Open Post Settings
 - Check if all tags are still there.

Update release notes:
